### ISSUE TITLE
Feature Request: BambuStudio plates for the whole pack

### ISSUE THREAD

**Issue by nerddotdad** [mr-kiou/q-pack#19]
Hello! Amazing work on all this it's incredible to see! I just went to etsy and purchased the guide as well! I'm wondering if anyone has put together build plates in a slicer like bambu studio, orca, or prusa slicer that can be downloaded in 1 shot. That would be incredible for a tired dad trying to print these for his kid.

**Comment by uurf** [comment]
It’s not a bad idea, but the plate sizes are specific to a printer, and with the various choices you have to make about which pack you are making, the combinations get big quickly.

**Comment by nerddotdad** [comment]
Totally fair after getting familiar with the doc after buying it! There are certainly a vast combination. I was hoping to do it by Halloween but it's definitely a more involved project for a sleep deprived dad 😂

**Comment by uurf** [comment]
Yeah I’ve done two packs, the second took less time than the first, but it would be a real accomplishment to print, assemble, finish, get all the greeblies etc a pack in 12 days. Even as a pretty accomplished 3d print hobbyist with a fast printer with a big print volume, a lot of these parts print times are measured in days. Get an early start on next year though!

**Comment by LostKatana** [comment]
I currently work on Bambu Lab Studio file for the P1S that I recently bough.
So the part size will be based on the L size.
I just started with it so it will definitely take some time and not only be one file I guess.
I try to label the plates according to the manual and the version to be printed.
However, I can not guarantee to really finish on this one.

**Comment by malake** [comment]
I've started on this today as a project for myself. I upgraded my printer to a Bambu from an AnkerMake/Eufy M5 and I'm really enjoying how convenient Bambu Studio makes it to pull in models and plan projects with multiple plates. I printed the shell of my first pack on my old printer and took an unplanned break from finishing. With cold weather setting in, it seems like as good of a time as any to finish what our ADHD started!

I understand that not everyone uses Bambu Studio or even the same printer as me, but through the collective power of open-source contributions I am optimistic it is possible to get to a place where the choices between packs printers are predictable. This can be accomplished with including feature branches, documentation, and git tagging.

Like I said, I'm starting this for myself today but will contribute back where I can when I'm finished.
